### DISCUSSION TITLE
Q: Bond Bridge not showing at BB18038.local

### DISCUSSION THREAD

**Discussion by brennanmh** [aarons22/homebridge-bond#141] [Q&A]
A: But it registered itself with my DHCP server as "expressif".

Real A: Best to just find the mac address, which is printed on the label on the bottom, and then in your router, if you have one, register that mac address for a static IP, as suggested.

It's also easy to do if you run your own dhcp server.

I received my bond bridge on March 8th, 2021 and it updated to a new firmware immediately, so things may have changed.

**Comment by aarons22** [comment]
Not sure if there is a question here, but as you noted, static IP is ideal for the best experience.
